graph/typeDefs.graphql

```diff
@@ -31,10 +31,10 @@ type User {
   username: String!
 
   # Action summaries against the user.
-  action_summaries: [ActionSummary]
+  action_summaries: [FlagActionSummary]
 
   # Actions completed on the parent.
-  actions: [Action]
+  actions: [FlagAction]
```

Comment:
This should be reverted, it breaks the composability that we had with the previous layout.